src/ui/build-navbar.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Copyright 2017-2019 The Last Pickle Ltd
```

Comment:
I guess we should now start putting Datastax copyrights on the files we touch.

Reply:
Raised issue #910 to fix this across the entire repo.